tgz2build/rules

```diff
@@ -82,7 +82,12 @@ $(STAMPDIR)/stamp-install: $(STAMPDIR)/stamp-build
 	$(INSTALL) -d -m 755 $(ZBS_STAGE_DIR)/$(ZBS_PREFIX)/share/man/man1
```

Comment:
Me neither, but I am not sure of it. But the files were there so I had to update them.